From caa80561b4acad9d66747ff6eb2b499ea67139e9 Mon Sep 17 00:00:00 2001
From: Ronny Chevalier <chevalier.ronny@gmail.com>
Date: Tue, 7 Apr 2015 19:35:13 +0200
Subject: [PATCH] zsh-completion: update loginctl

(cherry picked from commit 8470025541039f39391815b2ac93952003b7eee8)

Cherry-picked from: def740b
Resolves: #1222517
---
 shell-completion/zsh/_loginctl | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/shell-completion/zsh/_loginctl b/shell-completion/zsh/_loginctl
index 0de66e1..bd33b66 100644
--- a/shell-completion/zsh/_loginctl
+++ b/shell-completion/zsh/_loginctl
@@ -102,10 +102,11 @@ _arguments -s \
     '--kill-who=[Who to send signal to]:killwho:(main control all)' \
     {-s+,--signal=}'[Which signal to send]:signal:_signals' \
     {-H+,--host=}'[Operate on remote host]:userathost:_sd_hosts_or_user_at_host' \
-    {-M+,--machine=}'[Operate on local container]:machine' \
-    {-P,--privileged}'[Acquire privileges before execution]' \
+    {-M+,--machine=}'[Operate on local container]:machine:_sd_machines' \
     {-l,--full}'[Do not ellipsize output]' \
     '--no-pager[Do not pipe output into a pager]' \
     '--no-legend[Do not show the headers and footers]' \
     '--no-ask-password[Do not ask for system passwords]' \
+    {-n+,--lines=}'[Number of journal entries to show]' \
+    {-o+,--output=}'[Change journal output mode]:output modes:_sd_outputmodes' \
     '*::loginctl command:_loginctl_command'
